services/migrations/gitea_uploader.go

```diff
@@ -983,3 +880,25 @@ func (g *GiteaLocalUploader) Finish() error {
 	g.repo.Status = repo_model.RepositoryReady
 	return repo_model.UpdateRepositoryCols(g.repo, "status")
 }
+
+func (g *GiteaLocalUploader) remapExternalUser(source user_model.ExternalUserMigrated, target user_model.ExternalUserRemappable) (err error) {
+	userid, ok := g.userMap[source.GetExternalID()]
+	tp := g.gitServiceType.Name()
+	if !ok && tp != "" {
+		var err error
```

Comment:
```suggestion
```

I think this is redundant, because the return type already declares this variable name.

Reply:
Good catch, entirely unnecessary and removed.

---

[forgefriends source](https://lab.forgefriends.org/forgefriends/forgefriends/-/merge_requests/39#note_950)